api/scpca_portal/utils.py

```diff
@@ -26,6 +26,13 @@ def boolean_from_string(value: str) -> bool:
     return value.lower() in ("t", "true")
 
 
+def string_from_list(value: Union[List[Any], Any], delimiter=";") -> Union[str, Any]:
```

Comment:
```suggestion
def string_from_list(value: Any, delimiter=";") -> Any:
```
This typing is a bit confusing, I understand what you are going for here but really the function takes `Any` and returns `Any`. The doc string conveys that it converts lists to strings.

Reply:
See commit at deee8df44605e5447c7782630372196c5a7f4e2d.